### DISCUSSION TITLE
Export Google Sheets data to CSV

### DISCUSSION THREAD

**Discussion by japers** [AutomaApp/automa#249] [Q&A]
Thank you for this amazing software!

I am stuck trying to export a Google Sheets range to CSV. Below is a picture of my Workflow.

![image](https://user-images.githubusercontent.com/449872/148294184-cf51585e-47a9-4add-9b86-1c332de7e0da.png)

The G-Sheet is "Public" and the "Preview Data" in the Google Sheets Block correctly shows the data. However the CSV generated from this is blank. What am I doing wrong?

**Comment by Kholid060** [comment]
The **Export data** block only exports data inside the data columns, I will add more options in the future update.
